### PR TITLE
Implement Obsessive Stitcher, Primal Might, Siege Striker and...

### PR DESCRIPTION
**Pull request by Mikalinn777**
Waker of Waves.
Yeah that too.
Primal Might has weird wording. If someone wants to fix it, then be my guest.

### CODE REVIEW

**Comment by Mikalinn777** [comment]
I put them all as the same set for simplicity. I will update the other cards. Actually, would it be preferable if I set all three as different sets? If so, what set would I put the extra cards as?

**Comment by JayDi85** [comment]
@Mikalinn777 make two different PR -- one for new cards, one for historic format and set (different branches for each feature will help, don't use master).

**Comment by Mikalinn777** [comment]
@JayDi85 To be completely honest, I'm not sure how to take back the Historic Pull Request.
Secondly, do you think I should make a seperate set for each Historic Anthology, or one encompassing them all?
Thanks!

**Comment by JayDi85** [comment]
1. Just close that PR, create two different branches in your repository from commit before your changes ([see howto here](https://stackoverflow.com/a/18137009/1276632)). Copy and commit your code to that branches. Make PRs.
2. Yes, you must add different HA sets as scryfall. XMage must have scryfall set's structure (#6190).

**Comment by Mikalinn777** [comment]
Ok thank you.  How should I add the additional cards then? The ones that aren't in any specific historic anthology. (ex/ Rhys the Redeemed)

**Comment by JayDi85** [comment]
If you have multiple cards then you can add as one PR. I recommends:
1. PR for M21 cards;
2. PR for other cards;
3. PR for new sets and/or game formats.

The smaller PR, the faster it will be reviewed and merged.

**Comment by Mikalinn777** [comment]
Well I understand that lol. I'm asking for your advice on how to incorporate cards like Rhys the Redeemed into Historic. Rhys is not in any Historic Anthology set, and I'm not sure how else to incorporate it.

File: Mage.Server/config/config.xml [hunk 1]
```diff
@@ -160,6 +160,7 @@
         <deckType name="Constructed - Legacy" jar="mage-deck-constructed.jar" className="mage.deck.Legacy"/>
         <deckType name="Constructed - Vintage" jar="mage-deck-constructed.jar" className="mage.deck.Vintage"/>
         <deckType name="Constructed - Pauper" jar="mage-deck-constructed.jar" className="mage.deck.Pauper"/>
+        <deckType name="Constructed - Historic" jar="mage-deck-constructed-${project.version}.jar" className="mage.deck.Historic"/>
```
Comment (JayDi85) [inline Mage.Server/config/config.xml:163]: Don't need `{project.version}` in that file

**Comment by JayDi85** [comment]
> Well I understand that lol. I'm asking for your advice on how to incorporate cards like Rhys the Redeemed into Historic. Rhys is not in any Historic Anthology set, and I'm not sure how else to incorporate it.

I don't understand that you want to do. `Rhys the Redeemed` already implemented in xmage, see `Shadowmoor` set.

**Comment by Mikalinn777** [comment]
For the other Historic Anthology cards, I just made the set "Historic Anthology" legal. But the cards linked by emerald000 aren't in any specific set...

**Comment by JayDi85** [comment]
Ok, I've got what you mean. Single cards support in costructed formats added by latest commit: 642cdea2f896f38ee79097670a97945583aff422

Usage example:
shot_200612_002024

**Comment by Mikalinn777** [comment]
Ok thank you! I am Closing this PR now

**Pull request closed without merging**